app/views/users/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Here your users list</h1>
\ No newline at end of file
+<head>
+  <link rel="stylesheet" href="<%= asset_path('application.css') %>">
+</head>
+
+
+<% @users.each do |user| %>
+<div class="main-container">
+  <div class='user_img'>
+          <img src= <%= user.photo %>  />
+        </div>
+<div class="user-container">
```

Comment:
| Your User Index | Wireframe User Index |
| --- | --- |
| ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/75fb2de9-9cb2-46ad-8c91-6ca50f87c4ef) | ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/a1384174-c4ff-4135-8e6c-adbc1dc110ac) |

- [ ] It would be nice if you make the image size and the content size similar as you can see in the wireframe to make your project follow the requirements :+1: but now your image size is lot bigger than the expected size :fearful: